### ISSUE TITLE
[BUG]  An Error has Occured While Trying to Write Err.

### ISSUE THREAD

**Issue by GuyGieri** [charlesbel/Microsoft-Rewards-Farmer#164]
![image](https://user-images.githubusercontent.com/113925058/222863418-a590a62f-7c22-4533-b665-41bd549fec62.png)

Hey Alexgalaxy, i uncommented my username in line 20 and added my correct user but I still get this error

**Comment by DoShhPlease** [comment]
yes i also get this sometimes

**Comment by AlexyGalaxy** [comment]
When does this occur, What error is trying to be written? 
Did you fully set up your RewardsErr logger by chaning YourNameHere to your PC's Username?
```
'C://Users//YourNameHere//Desktop//Microsoft.Rewards.Err.txt'
```

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
